x-pack/plugins/enterprise_search/public/applications/app_search/components/app_search_gate/app_search_gate.tsx

```diff
@@ -0,0 +1,620 @@
+    }),
+  },
+  relevanceTuning: {
+    actionLabel: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.relevanceTuning.featureButtonLabel',
+      {
+        defaultMessage: 'Tune Search Relevancy',
+      }
+    ),
+    actionLink: 'https://www.elastic.co/guide/en/elasticsearch/reference/current/search-with-elasticsearch.html',
+    addOnLearnMoreLabel: undefined,
+    addOnLearnMoreUrl: undefined,
+    description: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.relevanceTuning.featureDescription',
+      {
+        defaultMessage:
+          "Tune the relevancy of your results using ranking and boosting methods",
+      }
+    ),
+    id: 'relevanceTuning',
+    learnMore: 'https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-boosting-query.html',
+    panelText: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.relevanceTuning.panelText', {
+      defaultMessage:
+        'Elasticsearch\'s query DSL provides an in-depth set of relevance tools.',
+    }),
+    title: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.relevanceTuning.featureName', {
+      defaultMessage: 'Relevance Tuning',
+    }),
+  },
+  curations: {
+    actionLabel: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.curations.featureButtonLabel',
+      {
+        defaultMessage: 'Use query rules',
+      }
+    ),
+    actionLink: 'https://www.elastic.co/guide/en/elasticsearch/reference/current/search-using-query-rules.html',
+    addOnLearnMoreLabel: undefined,
+    addOnLearnMoreUrl: undefined,
+    description: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.curations.featureDescription',
+      {
+        defaultMessage:
+          "Curate and pin results for specific queries",
+      }
+    ),
+    id: 'curations',
+    learnMore: 'https://www.elastic.co/blog/introducing-query-rules-elasticsearch-8-10',
+    panelText: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.curations.panelText', {
+      defaultMessage:
+        'Query rules provide a more robust set of tools to customize your search results for queries that match specific criteria and metadata.',
+    }),
+    title: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.curations.featureName', {
+      defaultMessage: 'Curate Results',
+    }),
+  },
+  searchManagementUis: {
+    actionLabel: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.searchManagementUis.featureButtonLabel',
+      {
+        defaultMessage: 'Build a search experience with Search UI',
+      }
+    ),
+    actionLink: 'https://www.elastic.co/docs/current/search-ui/overview',
+    addOnLearnMoreLabel: undefined,
+    addOnLearnMoreUrl: undefined,
+    description: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.searchManagementUis.featureDescription',
+      {
+        defaultMessage:
+          "Use user interfaces to manage your search and application experience",
```

Comment:
This seems a bit awkward, "Use user interfaces." Should it be reworded?

Reply:
That's the official copy from @leemthompo and @aznick approved so far in the doc...

Reply:
Made suggestion 👍